packages/actions/src/Imports/ImportColumn.php

```diff
@@ -116,6 +121,16 @@ public function example(mixed $example): static
         return $this;
     }
 
+    /**
+     * @param  array<mixed>  $examples
+     */
+    public function examples(array $examples): static
+    {
+        $this->examples = $examples;
+
+        return $this;
+    }
```

Comment:
This should be able to handle `mixed` instead of an array.

1) If the examples are an array or a Closure, set them directly as `$this->examples`
2) If the examples are not an array or a Closure, use `Arr::wrap()` and then set `$this->examples`